functions/src/venue.ts

```diff
@@ -598,36 +596,34 @@ export const upsertRoom = functions.https.onCall(async (data, context) => {
   admin.firestore().collection("venues").doc(venueId).update({ rooms });
 });
 
-export const deleteRoom = functions.https.onCall(async (data, context) => {
+export const deletePortal = functions.https.onCall(async (data, context) => {
   assertValidAuth(context);
 
-  const { venueId, room } = data;
+  const { spaceId, portal } = data;
 
-  const space = await getSpaceById(venueId);
+  const space = await getSpaceById(spaceId);
 
   await throwErrorIfNeitherWorldNorSpaceOwner({
-    spaceId: venueId,
+    spaceId,
     worldId: space.worldId,
-    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-    // @ts-ignore
-    userId: context.auth.token.user_id,
+    userId: context.auth?.token.user_id,
   });
 
-  const doc = await admin.firestore().collection("venues").doc(venueId).get();
+  const doc = await admin.firestore().collection("venues").doc(spaceId).get();
 
   if (!doc || !doc.exists) {
-    throw new HttpsError("not-found", `Venue ${venueId} not found`);
+    throw new HttpsError("not-found", `Venue ${spaceId} not found`);
   }
   const docData = doc.data();
   if (!docData) {
     throw new HttpsError("internal", `Data not found`);
```

Comment:
[NOTE]
Interesting, the above has `not-found` code, this one just `internal`. Might be an older line or copy-paste thing that makes them different. Might as well make them all `not-found`

Reply:
changed to "not-found": https://github.com/sparkletown/sparkle/pull/3000/commits/7051adca367c073619ecacf694eaa4bc4656cd4c